game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -63,7 +63,7 @@ class GameThread(scoreThread: ScoreThread, teamIDs: List[Int]) extends Runnable
   def updateGames: Unit = {
     for (game <- games) {
       val deltaTime = currentTime - previousTime
-      game.update(deltaTime)
+      game.update(deltaTime).foreach {scoreThread.addScoreModification(_)}
```

Comment:
I'm kind of teasing you, but I know what you like to put spaces more like `{ scoreThread.addScoreModification(_) }`